<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingpage</class>
 <widget class="QMainWindow" name="settingpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <!-- 向上移动所有按钮和标签 -->
   <widget class="QPushButton" name="Web">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>200</y>  <!-- 从260上移到200 -->
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/picture/Settingpage/dialogbutton.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Help">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>  <!-- 从260上移到200 -->
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/picture/Settingpage/dialogbutton.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Done">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>320</y>  <!-- 从320上移到260 -->
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/picture/Settingpage/dialogbutton.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   
   <!-- 隐藏滑块背景 -->
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="visible">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/picture/Settingpage/slider-track-2.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="visible">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/picture/Settingpage/slider-track-2.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   
   <widget class="QLabel" name="label_help">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>  <!-- 从270上移到210 -->
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Elephant&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_web">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>210</y>  <!-- 从270上移到210 -->
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Elephant&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Web</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_done">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>330</y>  <!-- 保持原位置 -->
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Elephant&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Done</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>271</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/picture/Settingpage/dialogtitle.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_settings">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Broadway&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
   <!-- 隐藏音量标签 -->
   <widget class="QLabel" name="label_volume">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>139</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="visible">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Elephant&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Volume</string>
    </property>
   </widget>
   <!-- 隐藏亮度标签 -->
   <widget class="QLabel" name="label_Bright">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="visible">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Elephant&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Brightness</string>
    </property>
   </widget>
   
   <!-- 语言切换按钮上移 -->
   <widget class="QPushButton" name="LangB">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>260</y>  <!-- 从260下移到320，放在Done按钮上方 -->
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/picture/Settingpage/Toggle_1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <!-- 语言标签上移 -->
   <widget class="QLabel" name="LangL">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>270</y>  <!-- 从270下移到330 -->
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Elephant&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>English</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>